pkg/api/frontendsettings.go

```diff
@@ -62,6 +63,11 @@ func (hs *HTTPServer) getFrontendSettings(c *contextmodel.ReqContext) (*dtos.Fro
 			continue
 		}
 
+		// Hide the datagrid until we remove the feature flag
+		if panel.ID == "datagrid" && !hs.Features.IsEnabled(featuremgmt.FlagEnableDatagridEditing) {
```

Comment:
This works great if we want to hide the datagrid alltogether. But having both this and the `PanelDataErrorView` seems a bit overkill.